cmd/reva/upload.go

```diff
@@ -76,9 +82,17 @@ func uploadCommand() *command {
 					Path: target,
 				},
 			},
+			Opaque: &typespb.Opaque{
+				Map: map[string]*typespb.OpaqueEntry{
+					"Upload-Length": {
```

Comment:
- [ ] Extend CS3APIS

Reply:
tracked in https://github.com/cs3org/cs3apis/issues/71